txtorcon/endpoints.py

```diff
@@ -404,7 +407,8 @@ def info_event(msg):
             self.config, self.hidden_service_dir,
             ['%d 127.0.0.1:%d' % (self.public_port, self.local_port)],
             group_readable=1)
-        self.config.HiddenServices.append(self.hiddenservice)
+        if len(self.config.HiddenServices) == 0:
```

Comment:
This isn't going to be sufficient; what if we connected to a Tor that already has 3 hidden-services?
We want to ensure that "our" hidden-service has been added.